wordpress-mobile/WordPress-iOS
# Share Extension gets configured regardless of .com connectivity

Fixes #6228

To test:
1. Reset the simulator.
2. Debug the share extension without having launched the main app. Verify warning pops up.
3. Debug the main app, do not log in.
4. Launch the share extension manually through the simulator and verify the warning pops up.
5. Log in with a .com site and verify the share extension works as expected.
6. Log out of the .com and then in with only a self-hosted site with no Jetpack connected. Verify the warning pops up.

Needs review: @jleandroperez

@sendhil this will need to be included in 6.7 if possible.
